specification/security_model.adoc

```diff
@@ -1,9 +1,195 @@
 [[security_model]]
-== Security and Threat Model
+== Security Model
+
+The CoVE-IO securiy model is built on the following assumptions:
+
+- Only the TVM owner can assess of a TDI trustworthiness. Based on that
+  assessment, it explicitly accepts or rejects a TDI into its TCB.
+- A TDI may access a TVM confidential memory only when the following conditions
+  are met:
+  1. The TVM owner has explicitly allowed the TDI to access its memory by
+     accepting it.
+  2. The TDI is exclusively assigned to the TVM, i.e. it must not be shared
+     other TVMs or any host software component.
+- A TVM may trust and access a TDI MMIO space if and only if the TDI is
```

Comment:
Here "TDI MMIO" means "trusted MMIO", right?
A device may have "untrusted MMIO", which does not follow this rule.

Reply:
Yes, but how would you define untrusted MMIO? A part of the TDI MMIO space that has not been mapped through the TSM controlled (and confidential memory hold) second page table (equivalent to secure EPT) ?

Reply:
According to TDISP spec, the device may return DEVICE_INTERFACE_REPORT for each MMIO_RANGE with bit IS_NON_TEE_MEM. 
* IS_NON_TEE_MEM = 0, means the MMIO is trusted MMIO.
* IS_NON_TEE_MEM = 1, means the MMIO is untrusted MMIO.

Reply:
Makes sense. Then we need to define that notion before using it.

Reply:
This should be clearer now.